resolwe/flow/executors/docker/run.py

```diff
@@ -115,26 +146,26 @@ def start(self):
         group_file.file.flush()
         self.temporary_files.append(group_file)
 
-        mappings.append({'src': passwd_file.name, 'dest': '/etc/passwd', 'mode': 'ro,Z'})
-        mappings.append({'src': group_file.name, 'dest': '/etc/group', 'mode': 'ro,Z'})
+        volumes.append({'src': passwd_file.name, 'dest': '/etc/passwd', 'options': 'ro,Z'})
+        volumes.append({'src': group_file.name, 'dest': '/etc/group', 'options': 'ro,Z'})
 
-        # create mappings for tools
+        # Create volumes for tools.
         # NOTE: To prevent processes tampering with tools, all tools are mounted read-only
         # NOTE: Since the tools are shared among all containers they must use the shared SELinux
         # label (z option)
-        self.mappings_tools = [{'src': tool, 'dest': '/usr/local/bin/resolwe/{}'.format(i), 'mode': 'ro,z'}
+        self.mappings_tools = [{'src': tool, 'dest': '/usr/local/bin/resolwe/{}'.format(i), 'options': 'ro,z'}
```

Comment:
This SELinux label will also be a problem on a shared storage.

Reply:
I left this as it was before? So this was already a problem before?

Reply:
Yes, it was already a problem before, I just pointed it out as the part for determining modes is being refactored.

Reply:
Good point, I'll add another extra option for `tools` and one for `passwd`/`group`.